tools/pip/setup.py

```diff
@@ -150,9 +150,8 @@ def skip_markdown_comments(md):
 package_data = {'mxnet': [os.path.join('mxnet', os.path.basename(LIB_PATH[0]))],
                 'dmlc_tracker': []}
 if variant.endswith('MKL'):
-    if platform.system() == 'Darwin':
-        shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/build/install/include'),
-                        os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
+    shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/build/install/include'),
```

Comment:
Curious how
1. previously when nightly CD was working, why was mkldnn include was done only for Darwin
2. now, to fix nightly CD, this needs to be done for all OS

Reply:
This platform condition was added by mistake in some earlier commit. We are fixing that here. No reason to package dnnl header files for only Darwin.